app/controllers/orders_controller.rb

```diff
@@ -10,11 +10,10 @@ def show
   end
 
   def create
-    # the four lines below probably would be best delegated to a GameCreator PORO
-    order = Order.new(user_name: "Rachel")
-    cart.contents.each do |donation_id, quantity|
-      order.donations.new(donation_id: donation_id, quantity: quantity)
-    end
+    # order = Order.new(username: "Rachel")
+    # cart.contents.each do |donation_id, quantity|
+    #   order.donations.new(donation_id: donation_id, quantity: quantity)
+    # end
```

Comment:
Because there is now a many-to-many relationship between the donations and the orders, this create method needs some work.

Also, why was this controller created before we had the Orders table created?